HomeWork 4/HomeWork 4/Program.cs

```diff
@@ -0,0 +1,105 @@
+﻿using System;
+
+class Program
+{
+    static void Main()
+    {
+        Console.Write("N = ");
+        int N = Convert.ToInt32(Console.ReadLine());
+        int[] arr = new int[N];
+        Console.WriteLine($"Enter {N} elements:");
+        for (int i = 0; i < N; i++)
+        {
+            Console.Write($"Element {i + 1}:");
+            arr[i] = Convert.ToInt32(Console.ReadLine());
```

Comment:
It's better to use Int32.TryParse(); as we can get an exception if this conversion fail